internal/scan/ruby.go

```diff
@@ -13,28 +13,7 @@ import (
 
 var RubyVersions []string = []string{"system"}
 
-func init() {
-	RubyVersions = append(RubyVersions, getRubyVersions()...)
-
-	if len(RubyVersions) == 1 {
-		log.Debug("rbenv not detected, falling back to system ruby ONLY. Please ensure that bundler is installed and available in your path.")
-		return
-	}
-
-	log.Debugf("Ruby versions detected: %+v\n", RubyVersions)
-
-	for _, version := range RubyVersions {
-		cmd := exec.Command("gem", "install", "bundler")
-		setRubyVersion(version, cmd)
-		data, err := cmd.CombinedOutput()
-		if err != nil {
-			log.Debugf("couldn't install bundler: %v", string(data))
-		}
-		log.Debugf("Installed bundler for ruby %v\n", version)
-	}
-}
-
-func getRubyVersions() []string {
+func GetInstalledRubyVersions() []string {
 	cmd := exec.Command("rbenv", "versions", "--bare")
 	data, err := cmd.Output()
 	if err != nil {
```

Comment:
Should this be returning RubyVersions instead when rbenv fails? E.g. so just returning "system"?

Reply:
Yes, good catch. I've also added a errorf log.